test/nodes/core/function/10-function_spec.js

```diff
@@ -1718,9 +1718,13 @@ describe('function node', function() {
     describe("init function", function() {
 
         it('should delay handling messages until init completes', function(done) {
+            const timeoutMS = 200;
+            // Since helper.load uses process.nextTick timers might occasionally finish
+            // a couple of milliseconds too early, so give some leeway to the check.
+            const timeoutCheckMargin = 5;
             var flow = [{id:"n1",type:"function",wires:[["n2"]],initialize: `
                 return new Promise((resolve,reject) => {
-                    setTimeout(resolve,200)
+                    setTimeout(resolve, ${timeoutMS});
```

Comment:
Ahhh - I didn't spot this was inside a `-string... was really confused as to how it wasn't being flagged as a syntax error. All good.